package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "js-miniapp",
+  "version": "0.1.0",
+  "description": "Mini App SDK for JavaScript",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "dependencies": {},
+  "devDependencies": {
+    "@istanbuljs/nyc-config-typescript": "^1.0.1",
+    "@types/chai": "^4.2.11",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^10.0.3",
+    "chai": "^4.2.0",
+    "gts": "^1.1.2",
+    "mocha": "^7.1.1",
+    "mocha-junit-reporter": "^1.23.3",
+    "nyc": "^15.0.0",
+    "sinon": "^9.0.1",
+    "source-map-support": "^0.5.16",
+    "ts-node": "^8.8.1",
+    "tslint": "^6.1.0",
+    "typedoc": "^0.17.3",
+    "typedoc-plugin-markdown": "^2.2.17",
+    "typescript": "^3.8.3"
+  },
+  "scripts": {
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "prebuild": "npm run clean && npm run check",
+    "build": "npm run clean && npm run check && tsc -p . && npm run test",
+    "fix": "gts fix",
+    "test": "mocha -r ts-node/register tests/**/*.spec.ts --reporter mocha-junit-reporter --reporter-options mochaFile=./test-results/mocha/results.xml",
+    "coverage": "nyc npm test",
+    "version": "npx -c 'echo $npm_package_version'",
+    "docs": "chmod +x scripts/docs.sh && scripts/docs.sh"
```

Comment:
You should be able to commit the `docs.sh` script as executable and then you don't need to have the `chmod` here.